project/src/pages/movie-page-screen/movie-page-screen.tsx

```diff
@@ -1,13 +1,43 @@
+import {useState} from 'react';
 import Header from '../../components/header/header';
 import Footer from '../../components/footer/footer';
+import {Films} from '../../types/film';
+import { Link, useParams } from 'react-router-dom';
+import { AppRoute, MoviePageState } from '../../const';
+import { FilmReviews } from '../../types/review';
+import FilmDetailsList from '../../components/film-details/film-details';
+import FilmReviewsList from '../../components/film-reviews/film-reviews';
+import FilmOverviewList from '../../components/film-overview/film-overview';
+
+type MoviesPageProps = {
+  films: Films;
+  filmReviews: FilmReviews[];
+}
+
+function MoviePage(props: MoviesPageProps): JSX.Element {
+  const params = useParams();
+  const [movieInfo] = props.films.filter((film) => film.id === Number(params.id));
+  const favoriteFilms = props.films.filter((film) => film.isFavorite);
+
+  const [state, setState] = useState('overview');
```

Comment:
:nit `[tab, setActiveTab]`

:nit - https://stackoverflow.com/questions/27810522/what-does-nit-mean-in-hacker-speak